README.md

```diff
@@ -99,3 +99,55 @@ FIREBASE_STORAGE_BUCKET=your-project-id.appspot.com
 Replace the content of `FIREBASE_SERVICE_ACCOUNT_KEY` with the JSON content from your service account key file (wrap the json in single quotes).
 
 The `FIREBASE_STORAGE_BUCKET` should be set to your Firebase Storage bucket name, typically in the format `your-project-id.appspot.com`.
+
+## Testing
```

Comment:
Ohh I think we need to mention the `yarn next:playwright-install` and `yarn next:cache-synpress` command right?